### PR TITLE
chore: add clear cache action

### PR DESCRIPTION
**Pull request by mainframev**

### CODE REVIEW

File: .github/workflows/clear-cache.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+name: Clear branch cache
+
+on:
+  workflow_dispatch:
+    inputs:
+      branch_number:
+        description: "Branch number"
```
Comment (DSil) [inline .github/workflows/clear-cache.yml:7]:
```suggestion
        description: "PR number"
```

The cache is associated with the PR number. So calling it PR number makes it clearer, imo

Comment (mainframev) [reply .github/workflows/clear-cache.yml]: agree, although it's kinda WIP :)

Comment (DSil) [reply .github/workflows/clear-cache.yml]: Yes, yes, it was not a formal review, just a suggestion

**Comment by mainframev** [comment]
@DSil this action seems to be not needed, as we can delete it manually in managements: 

Screenshot 2023-10-05 at 11 47 12

**Pull request closed without merging**